tree/master/cloud/src/solution/Microsoft.Legal.MatterCenter.Web/wwwroot/app/matter/createMatter.controller.js

```diff
@@ -2920,13 +2920,14 @@
                                             if (cm.isMatterDescriptionMandatory) {
                                                 var sCurrentMatterDesc = cm.matterDescription;
                                                 if (undefined !== sCurrentMatterDesc && null !== sCurrentMatterDesc && "" !== sCurrentMatterDesc) {
-                                                    sCurrentMatterDesc = sCurrentMatterDesc.trim(); bInValid = false;

```

Comment:
Removed the regular expression validation for Matter Description and now the matter description will allow all the special characters